pkg/platform/api/reqsimport/reqsimport.go

```diff
@@ -11,7 +11,7 @@ import (
 	"github.com/ActiveState/cli/internal/locale"
 	"github.com/ActiveState/cli/internal/logging"
 	"github.com/ActiveState/cli/pkg/platform/api"
-	"github.com/ActiveState/cli/pkg/platform/model"
+	"github.com/ActiveState/cli/pkg/platform/api/mono/mono_models"
```

Comment:
Changing this to avoid an import cycle when importing this file from the model package.